invenio_vocabularies/contrib/subjects/schema.py

```diff
@@ -23,13 +33,33 @@ class SubjectSchema(BaseVocabularySchema):
     # here.
     id = SanitizedUnicode(required=True)
     scheme = SanitizedUnicode(required=True)
-    subject = SanitizedUnicode(required=True)
+    subject = i18n_strings
```

Comment:
Do we want to switch this to `title` to be consistent with other vocabularies, or add the il8n_strings version as "title"? I think going from a string to il8n_strings is already a breaking change, so maybe we take the opportunity to get more consistency across vocabularies.

Reply:
Actually, [`title` already exists in `BaseVocabularySchema`](https://github.com/inveniosoftware/invenio-vocabularies/blob/810583f1354b997ce57101030a7631116e031802/invenio_vocabularies/services/schema.py#L90) as an `i18n_strings` field. In that case, @wgresshoff would it make sense that we leave `subject` as is, and instead populate translations in the `title` field? 

This also achieves the following:

- makes this change backward compatible (so no need to bump the subject schema/mappings to v2.0.0)
  - I only see new fields: `synonyms`, `identifiers` (is this still valid after [this comment](https://github.com/inveniosoftware/invenio-vocabularies/pull/332#discussion_r1647580257)?), and `title`
  - `title` needs to be added still to the JSONSchema and mappings thought
- we align with what we discussed with @SarahW91 today about making it possible for the administration UI to have a stable field to use as a localized display string